src/components/Label/index.js

```diff
@@ -27,23 +30,37 @@ const getSize = R.cond([
   [R.propEq('size', 'massive'), R.always(fonts.sizes.massive)],
 ]);
 
+const getBorderColor = R.cond([
+  [
+    R.both(R.propEq('inverted', true), R.propEq('primary', true)),
+    R.always(colors.primary),
+  ],
+  [
+    R.both(R.propEq('inverted', true), R.propEq('secondary', true)),
+    R.always(colors.secondary),
+  ],
+  [
+    R.both(R.propEq('primary', false), R.propEq('secondary', false)),
+    R.always(colors.grey),
+  ],
+  [R.T, R.always(colors.transparent)],
+]);
+
 const Label = styled.label`
    display: inline-block;
    line-height: 1;
    vertical-align: baseline;
    margin: 0em 0.14285714em;
-   background-color: ${getBackgroundColor};
+   background-color: ${props => props.inverted ? colors.white : getBackgroundColor};
    background-image: none;
```

Comment:
can u squish the logic how to set the background color for an inverted label into the `getBorderColor` function?